### ISSUE TITLE
Automatically use FaceID on password AutoFill

### ISSUE THREAD

**Issue by huantianad** [mssun/passforios#484]
Currently, pressing the autofill button on websites/apps will open pass, but still require a password/FaceID to authenticate (if you have that enabled). It would be nice if FaceID automatically activates if you do this, like on app startup.

**Comment by trizuz** [comment]
Yes. This would be a nice little feature. Have been thinking it about as well.
